tests/fireproof/fireproof.test.ts

```diff
@@ -45,7 +46,7 @@ describe("dreamcode", function () {
   });
   beforeEach(async function () {
     await sthis.start();
-    db = fireproof("test-db");
+    db = fireproof("test-db", { public: true });
```

Comment:
this is not the correct way to configure `public:true` but we should probably make this work instead of requiring the keyname trick